src/helpers/ApiCallHelper.tsx

```diff
@@ -1,7 +1,26 @@
+import { FaresQuery } from '../models/FaresQuery';
+
+const baseUrl = 'https://mobile-api-softwire2.lner.co.uk/v1/';
+
 export const fetchStations = () => {
-    return fetch('https://mobile-api-softwire2.lner.co.uk/v1/stations', {
+    return fetch(baseUrl + 'stations', {
         headers: {
             'X-API-KEY': `${process.env.REACT_APP_X_API_KEY}`,
         },
     });
 };
+
+export const fetchFares = (queryParams: FaresQuery) => {
+    return fetch(
+        baseUrl + 
+        'fares?originStation=' + queryParams.originStationCrs + 
+        '&destinationStation=' + queryParams.destinationStationCrs +
+        '&outboundDateTime=' + queryParams.outboundDateTime.toISOString() +
+        '&numberOfChildren=0' +
+        '&numberOfAdults=2', {
```

Comment:
we can specify these a bit neater using `URLSearchParams `like

```
    const params = new URLSearchParams({
        originStation: queryParams.originStationCrs,
        destinationStation: queryParams.destinationStationCrs,
        outboundDateTime: queryParams.outboundDateTime.toISOString(),
        numberOfChildren: '0',
        numberOfAdults: '2',
    });
```

Reply:
and then use this like `${baseUrl}fares?${params.toString()}`

in typescript using `${variable}text` ([template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)) to print you text is best practice